src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public abstract class Figure implements Calculation, Drawable {
+    private Color color;
+    private double area;
```

Comment:
we shouldn't store the area, we need to calculate it. So let's remove `double area` field